### PR TITLE
Include connection logic in top menu

### PR DESCRIPTION
**Pull request by MohamedHamouGisaia**

### CODE REVIEW

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 1]
```diff
@@ -78,6 +78,12 @@
 	"Login": "Login",
 	"No Account?": "Don't have an Account?",
 	"Sign up": "Sign up",
+	"You are about to launch process for": "You are about to launch process for",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:81]: launch _the_ process

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 2]
```diff
@@ -78,6 +78,12 @@
...
+	"product(s)": "product(s)",
+	"with the following parameters": "",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:83]: Missing translation

File: projects/arlas-toolkit/assets/i18n/en.json [hunk 3]
```diff
@@ -78,6 +78,12 @@
...
+	"Previous": "Previous",
+	"Validate": "Validate",
+	"Summary": "Summary",
 	"Session Expired": "Session Expired",
 	"You are no more connected or your session has expired. Would you like to log in again?": "You are no more connected or your session has expired. Would you like to log in again?",
```
Comment (QuCMGisaia) [inline projects/arlas-toolkit/assets/i18n/en.json:88]: no _longer_ connected

**Review by QuCMGisaia — APPROVED** [review]

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 2 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:83]: fixed

File: projects/arlas-toolkit/assets/i18n/en.json (hunk 3 above)
Comment (MohamedHamouGisaia) [reply projects/arlas-toolkit/assets/i18n/en.json:88]: fixed

**Pull request merged**